client/src/components/SignUp.js

```diff
@@ -118,7 +144,6 @@ function SignUp(props) {
             required
             type="text"
             error={!!errors.name}
-            helperText={errors.name}
```

Comment:
If we're taking out this type of validation, be sure to remove line 120 as well as all of the code from validations() related to these fields

Reply:
I edited it as you recommended. I could have removed the validations() step altogether (leaving only a value for errorConfirm), but I think I'd prefer to leave them in in case we want to add back in more custom errors or make the styling consistent, which would use validations().